playbooks/k8s-cluster/k8s-services.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- include: bootstrap-openshift.yml
+- include: ../bootstrap/bootstrap-openshift.yml
 
 # Install packages required for Ansible Helm module
 - hosts: kube-master
```

Comment:
I see we put the netapp-trident.yml in the k8s-cluster playbooks. Did you want to create a contrib directory?

Reply:
I wasn't sure how to break things up. What about `config.example/group_vars/netapp-trident.yml`? Should that go in a contrib directory and we document that you need to copy it out to your config directory?

Reply:
Do kubespray and kubeflow count as contrib since we added them?